feynman_path/diagram.py

```diff
@@ -175,10 +175,41 @@ def gate_arrow(self, g, time1, key1, key2, amp=1):
         xx2 = x2 - self.w_label/2 + self.arrow_off
         yy1 = y1 + (y2-y1)*(xx1-x1)/(x2-x1)
         yy2 = y2 - (y2-y1)*(x2-xx2)/(x2-x1)
-        color = '#26f'
-        if abs(float(amp) - abs(float(amp))) >= 1e-8:
-            color = '#e70'
+        #map the amplitude to  a "wheel" of colors
+        color = self.wheel_color(amp)
+        
         self.straight_arrow(g, color, xx1, yy1, xx2, yy2, width=w)
+    
+    def wheel_color(self,amp):
```

Comment:
Do you think this is enough colors for when you do Ry?  Alternatively, use some continuous color map like HSV where hue is controlled by the phase (i.e. [drawSvg.color.Hsv](https://github.com/cduck/drawSvg/blob/master/drawSvg/color.py))

Reply:
You are right, I was planning to update to a more continuous map later on when implementing the R* gates. I ll check the HSV